### PR TITLE
feat(blueprints router): add the router

### PR DESCRIPTION
**Pull request by cironunes**
WIP to address #66 

How far should we go with the implementation? I've tried to do as few as possible, but it would be nice to have at least one more route maybe showing the usage of `RouteParams`. WDYT?

### CODE REVIEW

**Comment by Brocco** [comment]
I like the idea of providing something with routing as part of the new, but am concerned that users may get annoyed of having to go in and delete a hello component every time they generate a new project.

Possibly adding the idea via a command arg to either turn that generation on or off?

**Comment by cironunes** [comment]
+1 to adding via command arg, or maybe prompting the user?

**Comment by Brocco** [comment]
Maybe both (arg + prompt) meaning we can prompt the user if the arg isn't specified? (not sure how I feel about it, but open for discussion)

Also, as for implementation, what about instead of adding the component to the ng2/new blueprint, how about calling the `ng generate component hello` from within to alleviate the maintenance on multiple blueprints?

**Comment by cironunes** [comment]
> Also, as for implementation, what about instead of adding the component to the ng2/new blueprint, how about calling the ng generate component hello from within to alleviate the maintenance on multiple blueprints?

Good catch. Will do

File: addon/ng2/blueprints/ng2/files/src/app.ts [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+  LocationStrategy
+} from 'angular2/router';
+
+
 import {<%= jsComponentName %>App} from './app/<%= htmlComponentName %>';
 
 
-bootstrap(<%= jsComponentName %>App);
+bootstrap(<%= jsComponentName %>App, [
+  ROUTER_PROVIDERS, provide(LocationStrategy, { useClass: HashLocationStrategy })]);
```
Comment (cironunes) [inline addon/ng2/blueprints/ng2/files/src/app.ts]: it should use PathLocationStrategy

**Comment by filipesilva** [comment]
@cironunes @Brocco are you happy enough with the implementation at https://github.com/angular/angular-cli/pull/139?

If so, I think this one can be closed.

**Pull request closed without merging**